app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/startup/CloudSdkVersionStartupCheck.java

```diff
@@ -30,6 +30,6 @@
   @Override
   public void runActivity(@NotNull Project project) {
     // If there is a configured Cloud SDK at this time, check that it is supported.
-    CloudSdkVersionNotifier.getInstance().notifyIfUnsupportedVersion();
+    CloudSdkVersionNotifier.getInstance().notifyIfVersionOutOfDate();
```

Comment:
same question if `notifyIfVersionParseError()` is applicable here?

Reply:
yes. good catch. ill add.